go/service/chat_local.go

```diff
@@ -511,6 +518,13 @@ func (h *chatLocalHandler) localizeConversation(
 		return chat1.ConversationLocal{}, err
 	}
 
+	// TODO need this more validation to make sure it's not a trick?
+	conversationLocal.Info.Usernames = utils.ReorderParticipants(
+		h.udc,
+		uimap,
+		conversationLocal.Info.TlfName,
+		conversationRemote.Metadata.ActiveList)
```

Comment:
It could, but the `ActiveList` can also be incomplete when not everyone has posted to the conversation. This way `Usernames` is always complete.